lxd/sys/apparmor.go

```diff
@@ -71,9 +71,9 @@ func (s *OS) initAppArmor() []cluster.Warning {
 
 	/* Detect AppArmor confinment */
 	profile := util.AppArmorProfile()
-	if profile != "unconfined" && profile != "" {
+	if profile != "unconfined" && profile != "" && !strings.HasSuffix(profile, "(unconfined)") {
 		if s.AppArmorAvailable {
-			logger.Warnf("Per-container AppArmor profiles are disabled because LXD is already protected by AppArmor")
+			logger.Warnf("Per-container AppArmor profiles are disabled because LXD is already protected by AppArmor via profile '%s'", profile)
```

Comment:
Please can we use `%q` here instead of `'%s'` thanks.

Reply:
Done in https://github.com/canonical/lxd/pull/12713/commits/54268e715567ccd42fb483193b967ce10a2bf6b0